client/src/entry/analysis/menu.js

```diff
@@ -242,19 +244,16 @@ export function fetchMenu(options = {}) {
             });
         }
         if (Galaxy.config.interactivetools_enable) {
-            userTab.menu.push({ divider: true });
             userTab.menu.push({
                 title: _l("Active InteractiveTools"),
                 url: "/interactivetool_entry_points/list",
             });
         }
-
-        // beta switch for activity bar
+        userTab.menu.push({ divider: true });
         userTab.menu.push({
-            title: _l("Show/Hide Activity Bar"),
-            onclick: () => {
-                userStore.toggleActivityBar();
-            },
+            title: _l("Sign Out"),
```

Comment:
I think language has left me behind here; I initially grated at the change but after checking gmail, github, and other accounts I think this is more modern phrasing, +1

Reply:
I noticed it in our preference page where we use "Sign Out" and when choosing between the two I also checked whatever I had opened and did not see "logged in" or "log out" once